src/main/java/org/jitsi/jigasi/transcription/WhisperConnectionPool.java

```diff
@@ -67,7 +62,7 @@ public WhisperWebsocket getConnection(String roomId)
             final WhisperWebsocket socket = new WhisperWebsocket();
 
             // connect socket in new thread to not block Smack threads
```

Comment:
Are we not worried about blocking Smack threads anymore? If not, update the doc :)

Reply:
It is done inside those methods.